build.sbt

```diff
@@ -102,7 +102,8 @@ lazy val akkaScalaNightly = akkaModule("akka-scala-nightly")
   .disablePlugins(ValidatePullRequest, MimaPlugin, CopyrightHeaderInPr)
 
 lazy val benchJmh = akkaModule("akka-bench-jmh")
-  .dependsOn(Seq(actor, actorTyped, stream, streamTests, persistence, distributedData, jackson, testkit).map(
+  .enablePlugins(Jdk9)
+  .dependsOn(Seq(actor, actorTyped, stream, streamTestkit, persistence, distributedData, jackson, testkit).map(
```

Comment:
`akka-jmh-bench` depended on `akka-stream-tests`, but used only one class from it. Moved that class to the `test` configuration of `akka-stream-testkit` instead, which probably wasn't strictly necessary, but seems a bit neater at least.